### ISSUE TITLE
Feature request: pushd/popd -q option

### ISSUE THREAD

**Issue by nfischer** [shelljs/shelljs#753]
Originally suggested in [this comment](https://github.com/shelljs/shelljs/issues/171#issuecomment-317327195) by @MattSturgeon

We might consider implementing the [`-q` option](http://zsh.sourceforge.net/Doc/Release/Shell-Builtin-Commands.html) (it's available in zsh, not in bash) for `pushd`/`popd`. this option does some zsh-specific things (i.e. the mention of `$chpwd_functions`), but it also suppresses console output.

In ShellJS, we would only want this to suppress console output (ignore the `$chwd_functions` stuff).

**Comment by alexreg** [comment]
Any progress on this lately?

**Comment by nfischer** [comment]
A PR would be welcome (and probably not very difficult). We only call `common.log()` [here](https://github.com/shelljs/shelljs/blob/master/src/dirs.js#L192) and [here](https://github.com/shelljs/shelljs/blob/master/src/dirs.js#L196). All that's needed is a little bit of plumbing to pass command options to `_dirs()`.

---

Workaround:

```js
shell.config.silent = true;
shell.pushd(...);
shell.config.silent = false;
```

**Comment by nfischer** [comment]
Fixed in #777
